src/main/kotlin/birdflop-presets.kt

```diff
@@ -0,0 +1,50 @@
+// This file adds the rgb.birdflop.com presets
+
+package chattore
+
+val birdflopGradients = mapOf(
+    "birdflop" to arrayOf(
+        "##084CFB",
+        "#ADF3FD"
+    ),
+    "SimplyMC" to arrrayOF (
+        "#084CFB",
+        "#ADF3FD"
+    ),
+    "Rainbow" to arrrayOF (
+        "#FF0000",
+        "#FF7F00",
+        "#FFFF00",
+        "#00FF00",
+        "#0000FF",
+        "#EAE4AA"
+    ),
+    "Skyline" to arrrayOF (
+        "#1488CC",
+        "#2B32B2"
+    ),
+    "Mango" to arrrayOF (
+        "#1488CC",
+        "2B32B2"
+    ),
+    "Vice City" to arrrayOF (
+        "#1488CC",
+        "#2B32B2"
+    ),
+    "Dawn" to arrrayOF (
+        "#1488CC",
+        "#2B32B2"
+    ),
+    "Rose" to arrrayOF (
+        "#1488CC",
+        "#2B32B2"
+    ),
+    "Firewatch" to arrrayOF (
+        "#1488CC",
+        "#2B32B2"
+    ),        
+)
+
+val birdflopPresets = birdflopGradients.mapValues { (_, colors) ->
```

Comment:
This is a little duplicated from the `pridePresets.kt`.

Reply:
Is that bad?

Reply:
I'd prefer to not have the same code repeated in 3 places. Since this is only really used in the ChattORESpec.kt, it could be added there as an extension function, potentially defined like this:

```kotlin
fun Map<String, Array<String>>.toGradientMinimessage() = this.mapValues { (_, colors) ->
    "<gradient:${colors.joinToString(':'.toString())}><username></gradient>"
}.toSortedMap()
```